### PR TITLE
Now handling package.json redirects

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: src/findEntryPoints.js [hunk 1]
```diff
@@ -3,9 +3,10 @@ var utils = require('./utils');
 
 function isValidFile (file, filePath, packageName) {
   return (
-    (path.extname(file) === '.js' || path.extname(file) === '.css') &&
+    (path.extname(file) === '.js' || path.extname(file) === '.css' || path.basename(file) === 'package.json') &&
     file[0] !== '_' &&
     file !== 'gulpfile.js' &&
+    !file.match(/webpack/) &&
```
Comment (CompuIves) [inline src/findEntryPoints.js:9]: Why is this one ignored?

File: src/utils.js [hunk 2]
```diff
@@ -118,6 +118,13 @@ module.exports = {
 
       return externals;
     }, {});
+
+    packageJsons.forEach(function (packageJson) {
+      console.log(packageJson);
```
Comment (CompuIves) [inline src/utils.js]: This one should be dropped maybe?

File: src/findEntryPoints.js (hunk 1 above)
Comment (christianalfoni) [reply src/findEntryPoints.js:9]: I saw that some packages has their webpack.config.js file in there, which might require Node packages that just breaks bundle or increases bundle size alot :-)

File: src/utils.js (hunk 2 above)
Comment (christianalfoni) [reply src/utils.js]: Ops :)

yes, let me fix

File: src/extractPackageJsonPaths.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            path: path.dirname(packageJson.replace(path.resolve(packagePath, 'node_modules') + '/', '')),
+            main: path.resolve(path.dirname(packageJson), parsedPackageJson.module || parsedPackageJson.main).replace(path.resolve(packagePath, 'node_modules') + '/', '')
+          }
+        })
+    }))
+      .then(function (packageJsonResults) {
+        return {
+          vendors: results.vendors,
+          packageJsons: packageJsonResults.filter(function (result) {
+            return Boolean(result)
```
Comment (CompuIves) [inline src/extractPackageJsonPaths.js:42]: Cool! I didn't know this trick 😄

File: src/findEntryPoints.js (hunk 1 above)
Comment (CompuIves) [reply src/findEntryPoints.js:9]: Wow, good find!

**Pull request merged**